Tests/Invoke-Tests.ps1

```diff
@@ -0,0 +1,54 @@
+<#
+.SYNOPSIS
+Runner to invoke Acceptance, Quality and / or Unit tests
+
+.DESCRIPTION
+Test wrapper that invokes
+
+.PARAMETER TestType
+[Optional] The type of test that will be executed. The parameter value can be either All (default), Acceptance, Quality or Unit
+
+.EXAMPLE
+Invoke-AcceptanceTests.ps1
+
+.EXAMPLE
+Invoke-AcceptanceTests.ps1 -TestType Unit
+
+#>
+
+[CmdletBinding()]
+Param (
+    [Parameter(Mandatory = $false)]
+    [ValidateSet("All", "Acceptance", "Quality", "Unit")]
+    [String] $TestType = "All"
+)
+
+$TestParameters = @{
+    OutputFormat = 'NUnitXml'
+    OutputFile   = "$PSScriptRoot\TEST-$TestType.xml"
+    Script       = "$PSScriptRoot"
+    PassThru     = $True
```

Comment:
This goes along with my original comment, we need to pick one 🙂

Reply:
I'll go through all of the scripts so far and ensure I enforce the new naming conventions.

Reply:
I've committed a newer version of Invoke-Tests.ps1 to enforce naming convention (see updated readme).